src/QtDcmFindScu.cpp

```diff
@@ -263,7 +238,7 @@ bool QtDcmFindScu::doQuery ( const OFList<OFString>& overrideKeys, QtDcmFindCall
                                 d->manager->currentPacs().port().toInt(),
                                 QtDcmPreferences::instance()->aetitle().toUtf8().data(),
                                 d->manager->currentPacs().aetitle().toUtf8().data(),
-                                UID_FINDPatientRootQueryRetrieveInformationModel, EXS_Unknown,
+                                queryRetrieveInfoModel.toStdString().c_str() , EXS_Unknown,
```

Comment:
Pour une prochaine itération, queryRetrieveInfoModel.toStdString().c_str() peut-être remplacé par:

- queryRetrieveInfoModel.toLatin1()
ou
- queryRetrieveInfoModel.toUtf8()